roles/setup_efm/tasks/setup_efm.yml

```diff
@@ -1,5 +1,28 @@
 ---
-# display details of node to install
+
+- name: Reference variables
+  include_vars: "{{ pg_type }}.yml"
+
+- name: Set the os variable
+  set_fact:
+    os: "{{ ansible_distribution | replace('Red Hat Enterprise Linux', 'RHEL') }}{{ ansible_distribution_major_version }}"
```

Comment:
Do you think we should use ansible_facts[‘os_family’] == 'RedHat' and make changes accordingly?

Reply:
Same as https://github.com/EnterpriseDB/edb-ansible/pull/85#discussion_r546015786